protos/sigstore_rekor.proto

```diff
@@ -106,7 +106,7 @@ message TransparencyLogEntry {
         InclusionPromise inclusion_promise = 5;
         // The inclusion proof can be used for offline or online verification
         // that the entry was appended to the log, and that the log has not been
-        // altered.
+        // altered. Also may be used as a signed timestamp.
```

Comment:
Possibly a misunderstanding on my part, but shouldn't this comment be on `inclusion_promise`, not `inclusion_proof`? Or does `inclusion_proof` also serve as a signed timestamp?

Reply:
Whoops, good catch, moved to `inclusion_promise`.